R/MsExperiment.R

```diff
@@ -0,0 +1,63 @@
+#'@include PlainTextParam.R
+#'@title Methods to save and load contents of a MsExperiment object
+#'
+#' @description
+#'
+#' For `MsExperiment`, the exported data and related text files are:
+#'
+#' - The backend's [spectraData()] stored in a tabular format in a text file
```

Comment:
I think it would be helpful (especially for external developers/users) if you would specify the content of the exported files. something like, *each row represents one sample with their metadata provided in the columns*. Maybe also for all the other data files you describe (e.g. also for the `MzBackendMzR` above. Describe them with the other developer in mind - if they see the first time such a file, what info would they need to understand its content and purpose.